src/annotations/selectors/index.ts

```diff
@@ -0,0 +1,35 @@
+// Types
+import {AppState} from 'src/types'
+import {sortBy} from 'lodash'
+
+// Mocks
+import {
+  MOCK_ANNOTATION_STREAMS,
+  AnnotationStream,
+} from 'src/annotations/constants/mocks'
+
+export const getAnnotationControlsVisibility = (state: AppState): boolean => {
+  return state.userSettings.showAnnotationsControls || false
+}
+
+export const getVisibleAnnotationStreams = (
+  state: AppState
+): AnnotationStream[] => {
+  const visibleStreams = state.annotations.visibleStreamsByID
+  const filtered = MOCK_ANNOTATION_STREAMS.filter(stream =>
+    visibleStreams.includes(stream.id)
```

Comment:
this feels expensive computationally (for every item in one array, go though all the items in another array). we should add visibility as client side meta data on the stream state. it might not come in from the api layer, but when we normalize the data, we append that default visibility state to it

Reply:
oh that sounds rad. Probably cleaner than having a whole different area in redux for it